### PR TITLE
Master staging

### PR DESCRIPTION
**Pull request by RazSherf**
Adding all the last changes

### CODE REVIEW

**Review by EkingIsrael — CHANGES_REQUESTED** [review]
Please sit with me at 14:35 @RazSherf 
We'll discuss this code review.

File: components/numberInput/NumberInputComponent.module.css [hunk 1]
```diff
@@ -1,5 +1,161 @@
...
+font-weight: 800;
+font-size: 16px;
+display: flex;
+flex-direction: row;
+gap: 7px;
+align-items: center;
+}
+
+.input_style{
+font-family:  Helvetica;
```
[the remark this answers is not in the document]
Comment (RazSherf) [reply components/numberInput/NumberInputComponent.module.css:19]: Ok, i changed it in the code 
this is the full font name :     font-family:  Arial, Helvetica, sans-serif;

File: components/numberInput/NumberInputComponent.jsx [hunk 2]
```diff
@@ -154,8 +157,8 @@ function NumberInputComponent({
           />
           {ShowControlButton ? (
             <div className={styles.inside_div_button}>
-              <button className={cx({ [styles.disabled]: Disabled }, [styles.plus_input_button])} type="button" onClick={addValue}>^</button>
-              <button className={cx({ [styles.disabled]: Disabled }, [styles.minus_input_button])} type="button" onClick={decreaseValue}>^</button>
+              <div className={cx({ [styles.disabled]: Disabled }, [styles.plus_input_button])} role="button" onClick={addValue} tabIndex={0} onKeyUp={() => {}}>^</div>
```
[the remark this answers is not in the document]
Comment (RazSherf) [reply components/numberInput/NumberInputComponent.jsx:160]: Ok. next time ill check with you before 👍🏼

**Comment by RazSherf** [comment]
adding the last changes

**Review by EkingIsrael — APPROVED** [review]

**Pull request merged**